pom.xml

```diff
@@ -42,6 +42,7 @@
 		<module>migrations</module>
 		<module>external/kafka</module>
 		<module>external/fixedWidth</module>
+		<module>examples</module>
```

Comment:
Didn't we discuss moving it to its own profile?

Reply:
Yup, notice the original conversation has not been marked as resolved yet. :-) (Together with the request to add a more complex example app)

Reply:
Done now (Let me know if the profiles could be done better, new to this)